src/azure-cli/azure/cli/command_modules/network/generated/commands.py

```diff
@@ -0,0 +1,47 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-statements
+# pylint: disable=too-many-locals
+# pylint: disable=line-too-long
+
+from azure.cli.core.commands import CliCommandType
+
+
+def load_command_table(self, _):
+
+    from ..generated._client_factory import cf_express_route_circuit_connection
+
+    network_express_route_circuit_connection = CliCommandType(
+        operations_tmpl='azure.mgmt.network.operations._express_route_circuit_connections_operations#ExpressRouteCircuitConnectionsOperations.{}',
+        client_factory=cf_express_route_circuit_connection,
+    )
+    with self.command_group(
+        'network express-route peering connection',
+        network_express_route_circuit_connection,
+        client_factory=cf_express_route_circuit_connection,
+    ) as g:
+        g.custom_command('list', 'network_express_route_peering_connection_list')
```

Comment:
Hi @qiaozha, How to setup min-api property by config?

Reply:
https://github.com/Azure/autorest.az#set-min-apimax-api-in-command-groupscommandsparameters